.eslintrc.js

```diff
@@ -32,7 +32,8 @@ module.exports = {
           "{}": false
         }
       }
-    ]
+    ],
+    "@typescript-eslint/no-namespace": "off"
```

Comment:
Should we allow to use namespace with the Typescript?
https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html

Reply:
I'm not sure about this, it's just used in the generated codes, not in our codes. so I think we can ignore this